crates/bins/src/bin/datadog-static-analyzer.rs

```diff
@@ -765,12 +776,16 @@ fn main() -> Result<()> {
             serde_json::to_string(&all_rule_results).expect("error when getting the JSON report")
         }
         OutputFormat::Sarif => {
-            let rules: Vec<SarifRule> = configuration
+            let static_rules_sarif: Vec<SarifRule> = configuration
                 .rules
                 .iter()
                 .cloned()
                 .map(|r| r.into())
                 .collect();
+
+            let secrets_rules_sarif: Vec<SarifRule> =
+                secrets_rules.iter().cloned().map(|r| r.into()).collect();
```

Comment:
```suggestion
            let secrets_rules_sarif: Vec<SarifRule> =
                secrets_rules.into_iter().map(|r| r.into()).collect();
```

We could avoid cloning by just consuming `secrets_rules` with `into_iter`, since we don't use it elsewhere in the function